src/graph_export/graph_export.cpp

```diff
@@ -51,8 +52,15 @@
 #endif
 namespace ovms {
 
+static std::string WriteOvmsVersionGraphLine(const std::string& projectName, const std::string& projectVersion) {
+    std::ostringstream oss;
+    oss << "# File created with: " << projectName + " " + projectVersion << std::endl;
+    return oss.str();
+}
```

Comment:
you dont need output string stream if all you do is concatenating string (+ camel case convention)
plus why are you mixing + and <<
```suggestion
static std::string writeOvmsVersionGraphLine(const std::string& projectName, const std::string& projectVersion) {
    std::stringstream ss;
    ss << "# File created with: " << projectName << " " << projectVersion << std::endl;
    return ss.str();
}
```

Reply:
now as I think of it, why do we need to construct it all the time? cant we use constexpr and save it in memory during compilation?

if thats not the case, cant we construct it once, save it in static const std::string and just oss <<  it everytime?